conf/application.conf

```diff
@@ -78,7 +78,7 @@ cache {
 # Microservice specific config
 
 auditing {
-  enabled = false
```

Comment:
I believe we will want to leave this turned off for local

Reply:
I would have thought that the config should have auditing switched on all the time.
We can override it for our local environments.
Is there a way of saying if localhost have false otherwise true?  like spring has in the .yaml files?
Also it will need to be switched on in order for this to be tested in any environment.

Reply:
This is essentially the config file for local. It is turned on in the other environments

Reply:
In that case yes it should be false, thank you

Reply:
Updated/put it back to false